js/core/version.js

```diff
@@ -1 +1 @@
-module.exports = "19.1.0"; /* DevExpress.VERSION */
+module.exports = "19.1"; /* DevExpress.VERSION */
```

Comment:
- Is the marker comment still required?
- Should we use placeholder `0.0.0` instead so we never need to patch it manually?

Reply:
Placeholder 0.0.0 is a good idea.